tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -130,6 +133,10 @@ spec:
         - "--effective-time=$(params.EFFECTIVE_TIME)"
         - "--output"
         - "yaml=$(params.HOMEDIR)/report.yaml"
+        # Write to both the new and the old deprecated result
+        # HACBS_TEST_OUTPUT will be removed in the future
+        - "--output"
+        - "hacbs=$(results.TEST_OUTPUT.path)"
```

Comment:
Oh we should probably change the name of the format to `test`? instead of it being `hacbs` ? It's visible to the user and it's going to be confusing.

Reply:
I did consider that, but a) I feel like "test" is a bad name for a format and b) it's not needed before the deadline, so we could postpone that and figure something out later.

Reply:
Split off to https://issues.redhat.com/browse/HACBS-2170 .